packages/dialect-react-ui/package.json

```diff
@@ -28,6 +28,7 @@
     "lint:fix": "yarn lint --fix"
   },
   "dependencies": {
+    "@cardinal/namespaces": "^1.1.0",
     "@cardinal/namespaces-components": "2.5.0",
```

Comment:
Please check if this package has newer version, I'm getting concerned with the amount of anchor packages we're installing :D